src/lib/urls.ts

```diff
@@ -1,2 +1,15 @@
 export const ofcomLicenseUrl =
   "https://www.ofcom.org.uk/manage-your-licence/radiocommunication-licences/ships-radio";
+
+export enum PageURLs {
+  environment = "/register-a-beacon/beacon-use",
+  purpose = "/register-a-beacon/purpose",
+  activity = "/register-a-beacon/activity",
+  landOtherActivity = "/register-a-beacon/land-other-activity",
+  aboutTheVessel = "/register-a-beacon/about-the-vessel",
+  aboutTheAircraft = "/register-a-beacon/about-the-aircraft",
+  vesselCommunications = "/register-a-beacon/vessel-communications",
+  aircraftCommunications = "/register-a-beacon/aircraft-communications",
+  landOtherCommunications = "/register-a-beacon/land-other-communications",
+  moreDetails = "/register-a-beacon/more-details",
+}
```

Comment:
Create initial set of `PageURLs` relevant to what I was testing in this PR.  Propose that we gradually migrate our URL references over to this enum over time.

Reply:
👍  Suggestion: extract the `register-a-beacon` as a const?

Reply:
@matthew-a-carr Ignore comment above.  String interpolation not possible in Typescript enums and favour repeating string for readability